config/jobs/kubernetes/sig-windows/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
```

Comment:
At some point there will be test jobs other than the windows-gce.yaml ones under this directory. Suggestions for now: leave pjh, ask yujuhong if they want to be included or not, leave yourself out since you haven't participated much in sig-windows yet :/

Then when other folks from sig-windows move test configs into this directory they can add themselves to this owners file in the same PR.

Reply:
yujuhong doesn't want to be included, so removed us

Reply:
I would expect to see some combination of SIG Chairs/Technical Leads here.
[As of writing](https://github.com/kubernetes/community/blob/cf6db53e3104d7dbf4c3f13f172483a09ff52a00/sig-windows/README.md), the following contributors are listed as leads:
- @PatrickLang # SIG Windows Chair
- @michmike # SIG Windows Chair
- @benmoss # SIG Windows Technical Lead
- @ddebroy # SIG Windows Technical Lead